algorithmiaio/langpacks
# Added H2O python client support

* updated the java11 package to remove unnecessary gpg keyserver exchange (which fails consistently and breaks stuff)
* added a h2o-python template
* to verify, lines 11, 13-21 have to be removed from /languages/Dockerfile.compile.j2 in advance
verification input: `python3 tools/environment_validator.py -g python3 -s python37 -d java11 -t dependency -n h2o-python`
verify with:  `curl localhost:9999 -H 'Content-Type: application/json' -d '"algorithmia"'`
**edit**: now uses ubuntu:20.04 instead of 16.04 as our base image to better facilitate java. This change now effects Java.
To test h2o/python:
`python3 tools/environment_validator.py -g python3 -s python37 -d java11 -t dependency -n h2o-python`
To test java11:
`python3 tools/environment_validator.py -s java11 -t language -n java11`

respond with the following in a separate tab to verify algo operations:
`curl localhost:9999 -H 'Content-Type: application/json' -d '"algorithmia"'`
